engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/new/views/new-repository.html

```diff
@@ -274,7 +274,7 @@ <h4 translate> Sync Settings </h4>
           </p>
         </div>
         <div ng-if='option.input_type=="textarea"' bst-form-group label="{{ option.title | translate }}">
-	  <textarea
+	        <textarea
```

Comment:
Looks like this got edited accidentally:
```suggestion
	  <textarea
```

Reply:
This was actually intentional, looked like this in VSCode for me:
![image](https://github.com/user-attachments/assets/afc240b7-2b4e-4419-8ee5-29ba868a862f)

But yes, it's unnecessary I will remove it again 👍🏻